radish34/ops/await_stack.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# TODO: wait for all services...
+./ops/await_tcp.sh -t 30 -h localhost -p 8080
```

Comment:
I can't help but think we could use dockers built in health check and service dependency graph feature here. What are your thoughts on that? We have a ticket for that approach https://github.com/ethereum-oasis/baseline/issues/51

Reply:
And potentially some features of swarm.

Reply:
Very much open to that.

This `await_tcp` approach is solid with the exception of needing to add a fallback if `which timeout` fails (will push that fix shortly to make it less annoying for everyone).

Less code to maintain is always better than more. I'd be happy to adopt the docker health check approach you suggest before we merge this.

Will look into #51 in the meantime.